File: Codewars/8KYU_Swift_JavaScript/19_bmi/bmi.swift
/*
Write function bmi that calculates body mass index (bmi = weight / height2).

if bmi <= 18.5 return "Underweight"

if bmi <= 25.0 return "Normal"

if bmi <= 30.0 return "Overweight"

if bmi > 30 return "Obese"
*/

func bmi(_ weight: Int, _ height: Double) -> String {
    let bmiValue = Double(weight) / (height * height)
    switch bmiValue {
        case ...18.5: return "Underweight"
        case 18.5...25.0: return "Normal"
        case 25.0...30.0: return "Overweight"
        case 30.0...: return "Obese"
        default: return "wrong input"
    }
}

print(bmi(80, 1.6))

File: Codewars/8KYU_Swift_JavaScript/19_bmi/bmi.js
/*
Write function bmi that calculates body mass index (bmi = weight / height2).

if bmi <= 18.5 return "Underweight"

if bmi <= 25.0 return "Normal"

if bmi <= 30.0 return "Overweight"

if bmi > 30 return "Obese"
*/

function bmi(weight, height) {
    const bmiValue = weight / (height * height);
    switch (true) {
        case bmiValue <= 18.5: return "Underweight";
        case bmiValue <= 25.0: return "Normal";
        case bmiValue <= 30.0: return "Overweight";
        case bmiValue > 30.0: return "Obese";
    }
  }

  console.log(bmi(70, 1.6));